requirements/django.txt

```diff
@@ -8,8 +8,8 @@ django-jsonfield==0.9.13
 django-countries==2.1.2
 git+http://github.com/cdr-stats/django-pagination.git
 django-admin-tools==0.7.2
+django-dajaxice-ng==0.5.6.4
```

Comment:
we could maybe stick with 0.7.0.2, seems like the change was introduced at that time.